spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/ZipkinSpanListener.java

```diff
@@ -44,6 +47,24 @@
  * @since 1.0.0
  */
 public class ZipkinSpanListener {
```

Comment:
If stream depends on this, make a utility and share it.. otherwise, I understand why c&p, just carry over the same comments here.